### ISSUE TITLE
Please have a look

### ISSUE THREAD

**Issue by DorianPuscas** [ObsessiveCompulsiveAudiophile/AudysseyOne#4]
So i did all the 8 measurements .
I have 7.1 sistem , front speakers are floorstanders , older model Canton Karat - subwoofer on the side of the box , so capable speakers .
Receiver Denon X3500H .
The idea is Original Audissey sets the front speakers Large .
The optimized Audissey sets speakers to small and crossover at 250 Hz , dont know why and is this the best option ?
Also the subwoofer is set at -12Db level , so the maximum minim for the sub , dont know why .

I am trying to put the measurements here, maybe you can have a look please .
Also the measurement of the subwoofer before and after - original and optimized .

Maybe you can advise me what to change , or leave it as it is .

Thank you very much .
It is attached as ZIP .
[Original OK.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14641681/Original.OK.zip)

**Comment by DorianPuscas** [comment]
I also have REW and UMIK 1 if i can help with something .

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Thanks. Crossover calculation was based on correlation with 24dB/oct roll off Linkwitz Riley slopes and these regularly detect major dips as crossovers higher up the freq band. I have revised the XO calculation mechanism completely in today's update (v1.1 here,). WIth multiple subs I could understand if there was a problem but with a single sub volume alignment works perfectly. Your sub is probably running too hot.

**Comment by DorianPuscas** [comment]
Hmm , when i took the measurements it was in the green barrier (you know when it sets the volume of the sub in the app) .
Will try again with new version and let you know if something changes
.

**Comment by DorianPuscas** [comment]
Ok, so i uploaded my measurements in the new version. Here are the results , maybe it will help you .
Thank you very much and appreciate your help . 
If its no offense, i will follow and keep you updated .

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Don't see a link?

**Comment by DorianPuscas** [comment]
[Denon OCA.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14642693/Denon.OCA.zip)

**Comment by DorianPuscas** [comment]
So the sound is amazing , i only tested for movie Atmos and Dolby. The effects are really really unbelievable .
BUT :) 
dont get upset (angry), i have literally no bass , so i have to bump the AVR to +12db subwoofer .
My sub volume is allmost at half, but is really strong at this volume regularly 

Dont get it , the sub is allmost nonexistent , when i bump it to 12 db , then i can hear it a little bit . 
I have LFE + Main 120 Hz set

**Comment by DorianPuscas** [comment]
I dont know if here is the right place for this , but i also have a SMSL SU1 connected to Denon X3500H on rca . So from pc to SMSL to AVR Rca , i only listen 2.1 in this configuration . 
I have REW and Umik 1 measurements , separate only front no sub , front with sub , only sub . If its not much trouble with your permission i would sent them to have a look, just if its no big deal to you , maybe youre busy , maybe another time .

I will keep you update if it helps

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Your subwoofer volume was set to -12dB and set to small with a crossover at
40Hz during the calibration process if your original.ady file is any
reference. None of these are normal outcomes. You should run the
calibration properly or replace your calibration mic if it were showing
green during calibration as you have stated.

**Comment by DorianPuscas** [comment]
Understood . My daughter is sleeping now, but tomorrow i will make new measurements from start and see if anything changes . I will keep you posted

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Sure thing. Good night.

**Comment by ba5e2k** [comment]
@ObsessiveCompulsiveAudiophile had a look through the new changes in the html, see a fair bit of new code, can you post a short changeling for the new version you released yesterday?

**Comment by DorianPuscas** [comment]
Hello again , i did a new measurement of all 8 positions and made everything new with the lastest version (19.03.2024 - 22:30 PM) . i am attaching everything including a photo of sub level when starting Audissey EQ .
[19.03.2024.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14656349/19.03.2024.zip)

**Comment by ObsessiveCompulsiveAudiophile** [comment]
I have just updated again now (to v1.2). Yesterday's XO matching algo was
failing with many systems and required too much memory. Now, it's all good.
I uploaded it to github already.

**Comment by DorianPuscas** [comment]
System Log:
11:17:22 PM [INFO] Based on your AVR model - Denon AVR-X3500H, speed of sound has been set to 300.0 m/s for all distance calculations.
11:17:22 PM [WARNING]  - if you believe this setting is not correct for your receiver model, please notify OCA!
11:17:22 PM [INFO] ************************************************************************************************************************
11:17:22 PM [LOG] Applying Target Curve located @ C:\Users\doria\Downloads\DrTooleTargetCurve.txt
11:17:22 PM [INFO] ************************************************************************************************************************
11:17:22 PM [INFO] Optimization process started...
11:17:22 PM [LOG] Transposing measurements from '8' different microphone positions to the primary listening position...
11:17:28 PM [INFO] Done!
11:17:28 PM [INFO] ************************************************************************************************************************
11:17:28 PM [INFO] Cleaning up processed measurements...
11:17:42 PM [LOG] Done!
11:17:42 PM [INFO] ************************************************************************************************************************
11:17:42 PM [LOG] Calculating precise speaker volume levels...
11:17:43 PM [LOG] Co: 74.83510200977327 dB
11:17:43 PM [LOG] FDLo: 73.98010282516479 dB
11:17:43 PM [LOG] FDRo: 69.40905981063842 dB
11:17:43 PM [LOG] FLo: 71.59397578239441 dB
11:17:44 PM [LOG] FRo: 71.07822663784027 dB
11:17:44 PM [LOG] SLAo: 79.13670778274536 dB
11:17:44 PM [LOG] SRAo: 80.5238921403885 dB
11:17:44 PM [INFO] Optimized target volume level for all speakers: 74.365295284135 dB
11:17:44 PM [INFO] Total number of speakers averaged (excludes subwoofer): 7
11:17:44 PM [LOG] Finalized speaker volume settings:
11:17:44 PM [LOG] C: -0.5 ; with 0.03019327436173569 dB filter compensation
11:17:44 PM [LOG] FDL: 0.5 ; with -0.11480754102979063 dB filter compensation
11:17:44 PM [LOG] FDR: 5.0 ; with -0.04376452650342344 dB filter compensation
11:17:44 PM [LOG] FL: 3.0 ; with -0.22868049825940773 dB filter compensation
11:17:44 PM [LOG] FR: 3.5 ; with -0.21293135370527239 dB filter compensation
11:17:44 PM [LOG] SLA: -5.0 ; with 0.22858750138964012 dB filter compensation
11:17:44 PM [LOG] SRA: -6.0 ; with -0.15859685625349584 dB filter compensation
11:17:45 PM [INFO] Optimizing subwoofer volume adjustment...
11:17:45 PM [ERROR] Required subwoofer volume adjustment is beyond AV Receiver's hardware limits!
11:17:45 PM [WARNING] Be advised to repeat Audyssey calibration process after re-adjusting subwoofer volume!
11:17:45 PM [INFO] The algorithm will now apply maximum possible volume setting to the subwoofer and continue.
11:17:45 PM [WARNING] However, optimization will not result in the best achiveable performance!
11:17:45 PM [LOG] SW1: -12.0 dB ; with 0 dB filter compensation
11:17:45 PM [INFO] ************************************************************************************************************************
11:17:45 PM [INFO] Deriving optimal speaker distances with advanced cross-correlation analysis...
11:17:46 PM [LOG] C: 2.55m
11:17:46 PM [LOG] FDL: 4.99m
11:17:46 PM [LOG] FDR: 4.75m
11:17:46 PM [LOG] FL: 3.27m
11:17:46 PM [LOG] FR: 2.87m
11:17:46 PM [LOG] SLA: 1.93m
11:17:46 PM [LOG] SRA: 2.74m
11:17:46 PM [LOG] SW1: 6.14m
11:17:46 PM [INFO] Subwoofer distance is 'optimally' set @ 6.13m and is 'safely' within the AV Receiver's speaker spacing limits.
11:17:46 PM [INFO] ************************************************************************************************************************
11:17:46 PM [INFO] Crafting custom room correction filters...
11:17:50 PM [LOG] Optimal filter generation for speaker C is completed.
11:17:54 PM [LOG] Optimal filter generation for speaker FDL is completed.
11:17:58 PM [LOG] Optimal filter generation for speaker FDR is completed.
11:18:02 PM [LOG] Optimal filter generation for speaker FL is completed.
11:18:06 PM [LOG] Optimal filter generation for speaker FR is completed.
11:18:10 PM [LOG] Optimal filter generation for speaker SLA is completed.
11:18:14 PM [LOG] Optimal filter generation for speaker SRA is completed.
11:18:18 PM [LOG] Optimal filter generation for speaker SW1 is completed.
11:18:19 PM [LOG] Done!
11:18:19 PM [INFO] ************************************************************************************************************************
11:18:19 PM [INFO] Pinpointing crossover frequencies for each speaker...
11:18:20 PM [ERROR] Error: Speaker has no bass response! - program execution has stopped!

**Comment by DorianPuscas** [comment]
Same thing here too about my sub ... and an error at the end

11:17:45 PM [INFO] Optimizing subwoofer volume adjustment...
11:17:45 PM [ERROR] Required subwoofer volume adjustment is beyond AV Receiver's hardware limits!
11:18:20 PM [ERROR] Error: Speaker has no bass response! - program execution has stopped!

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Can you share your ady that caused that?

**Comment by DorianPuscas** [comment]
[Orig 19.03.2024.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14656782/Orig.19.03.2024.zip)

**Comment by ObsessiveCompulsiveAudiophile** [comment]
This is a normal Audyssey calibration file data:

**Comment by DorianPuscas** [comment]
yes , i dont know what to send you ...:(
I uploaded this in new version and then i got this error , the program never finished to have a final ady to send to you , it gave that error and stopped

**Comment by ObsessiveCompulsiveAudiophile** [comment]
This is a normal Audyssey calibration data imported to REW like the
hundreds I have in my computer

[image: image.png]

and this is your calibration data:

[image: image.png]

Even the noise level of the subwoofer is the same as the speaker levels.I
am trying to find out what is causing this...

**Comment by DorianPuscas** [comment]
i can send you my measurements made with REW and UMIK 1 if you think it helps you . only fronts and sub measured

**Comment by ObsessiveCompulsiveAudiophile** [comment]
No, it's not gonna help. Audyssey will not measure something 40dB higher
than the others. That's 6 times the volume. Obviously, the script finds no
speaker response at these levels and throws an error. How did you measure
this? WHat app version? Android/Ios?

**Comment by DorianPuscas** [comment]
Android - version 1.11.0

Same as yesterday, and like an hour ago .
It worked until now

**Comment by ObsessiveCompulsiveAudiophile** [comment]
It didn't work, the older didn't check for bass response properly and
dimmed your SW volume by the limit of -12dB and finished. Your sub needs
-42dB volume adjustment according to your ady responses!!!

**Comment by DorianPuscas** [comment]
How to do this , when i open the audiseey eq , at the beggining it says it will adjust the subwoofer level and then i have that red and green line , i adjust the volume knob from sub to meet the green line , like in this picture 
[Screenshot_20240319_213005_com.dmholdings.AudysseyMultEq.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14656954/Screenshot_20240319_213005_com.dmholdings.AudysseyMultEq.zip)

**Comment by ObsessiveCompulsiveAudiophile** [comment]
I think either your app is corrupt or something is wrong in your mic. Subwoofer volume comes out by default 40dB louder than the rest of the speakers. Best I can do it is to dim it down by -12 dB and it's still 30dB above the speakers. Let me put it this way, If your sub had really been as loud as it shows in the graphs, you would be instantly deaf when it plays. 130dB is jet engine from 1m. So, definitely something goes wrong during the measurement process.

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Yes, that's how it's done and even if you left it all the way in the red,
it would still not be that loud as it comes up in the ady file. Try
re-installing your app maybe.

**Comment by DorianPuscas** [comment]
Something is wrong, because the sub is hardly audible even when measuring , and afterwards the same - low level sub .
The mic if it were to be faulty , the speakers should have some errors or something when measuring maybe ? 
Dont know ... I will install another version of the app , if i cand find one older maybe . I dont think the mic is the problem and i dont have another besides the UMIK 1 - usb .

**Comment by DorianPuscas** [comment]
Just a second , maybe if you have a look of the subwoofer measured with UMIK in REW , maybe you will see something there , worth a try and your keen eye may spot my problem 
[OCA to send FR FL Sub UMIK 1.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14657297/OCA.to.send.FR.FL.Sub.UMIK.1.zip)

**Comment by ObsessiveCompulsiveAudiophile** [comment]
I can make a calibration file from your measurements for you but I need
some time. I am a bit swamped with work and the channel these days (I have
a day time job). Your ady files are not being produced right. Noone can fix
them. I have to randomly lower the volume of the sub and start calibration
from there but it will be meaningless.

**Comment by ObsessiveCompulsiveAudiophile** [comment]
Yes, in these measurements the subwoofer response is where it should be.

**Comment by DorianPuscas** [comment]
Hmm , then i think i will wait for your help on this .
If you can do something for my problem i highly appreciate you .

I will wait for news from you then .
Thank you so much

**Comment by ObsessiveCompulsiveAudiophile** [comment]
https://drive.google.com/file/d/1tgDl8CeFI6nO63AkIf4EKToMDBy39XdW/view?usp=sharing

Here's your optimized ady file. I dropped sub volume by -42dB (this is a
common SPL offset in maths and subs seemed to be close to the
levis relative to FL and FR based on your REW measurements) and started the
optimization from there. I guess you can adjust the sub volume by +,- 3dB
and find the right level. The rest is already optimized.

**Comment by DorianPuscas** [comment]
WAW , youre the best . Thank you so so much, really appreciate it !
Will try it tomorrow on day time and let you know how it sounds . Thank you

And have a good night

**Comment by ianfretwell** [comment]
Just a quick question to @DorianPuscas on this issue  -  you are doing the measurements with the Audyssey mic aren't you - and NOT the UMIK1 ?

**Comment by DorianPuscas** [comment]
Yes , with the audisseyEQ app conmected to Avr and then Avr measured with standard mic , provided with the avr. The umik 1 has usb

**Comment by ianfretwell** [comment]
I know but it just seems strange that you're having all these issues!

**Comment by DorianPuscas** [comment]
Sorry for late reply. Dont understand why it is happening. 
I must settle for now with Oca's ady file he made . 
I think is a problem from the multeq app , when i measure the normal way with the app from the receivernthe subwoofer is set to +1 db , and with the mobile app -12 db

**Comment by michaelbarkan** [comment]
Many versions of the mobile app have the -12dB sub problem. Here is the old version before the problem originated:
https://drive.google.com/file/d/1Xz1Hb-nIHSCpeO9YBW4zarpKkcaRLUwu/view?usp=sharing
Michael

**Comment by DorianPuscas** [comment]
Michael , thank you for this version. I uninstalled the new one and installed this one. Will take the measurements with this one (older) and see how it works . Will write here off the changes

**Comment by DorianPuscas** [comment]
Finally, been able to measure and the sub is in normal volume level for tuning . Thank you for the version, i looked for it but couldnt find it myself until now .
[25.03.2024 orig multeq old.zip](https://github.com/ObsessiveCompulsiveAudiophile/AudysseyOne/files/14735528/25.03.2024.orig.multeq.old.zip)

**Comment by michaelbarkan** [comment]
Great news. Glad to have been able to help.

**Comment by DorianPuscas** [comment]
Thanks again very much . I now have awesome sound , the measurement waas made correct with the older version . 
OCA is the best , he made a remarkable solution for us , home theater guys :)
